### PR TITLE
Add TimeUnit null check test case in Timed

### PR DESCRIPTION
**Pull request by ggikko**
1. It's about Timed.java test case that is null check TimeUnit parameter.
2. formatting in BasicIntQueueDisposable
3. add testcase blockingIterableNegativeBufferSizeFail for blockingIterable
4. modify BlockingMultiObserver field’s modfier

### CODE REVIEW

File: src/main/java/io/reactivex/internal/observers/BasicIntQueueDisposable.java [hunk 1]
```diff
@@ -22,10 +22,8 @@
  * that defaults all unnecessary Queue methods to throw UnsupportedOperationException.
  * @param <T> the output value type
  */
-public abstract class BasicIntQueueDisposable<T>
-extends AtomicInteger
-implements QueueDisposable<T> {
-
+public abstract class BasicIntQueueDisposable<T> extends AtomicInteger
+        implements QueueDisposable<T> {
```
Comment (JakeWharton) [inline src/main/java/io/reactivex/internal/observers/BasicIntQueueDisposable.java]: Revert style changes

File: src/main/java/io/reactivex/internal/observers/BlockingMultiObserver.java [hunk 2]
```diff
@@ -27,10 +27,10 @@
 extends CountDownLatch
 implements SingleObserver<T>, CompletableObserver, MaybeObserver<T> {
 
-    T value;
-    Throwable error;
+    private T value;
+    private Throwable error;
 
-    Disposable d;
+    private Disposable d;
```
Comment (JakeWharton) [inline src/main/java/io/reactivex/internal/observers/BlockingMultiObserver.java]: Revert visibility changes. These are intentional

File: src/main/java/io/reactivex/internal/observers/BasicIntQueueDisposable.java (hunk 1 above)
Comment (ggikko) [reply src/main/java/io/reactivex/internal/observers/BasicIntQueueDisposable.java]: yeah, I changed :)

File: src/test/java/io/reactivex/internal/operators/observable/ObservableBlockingTest.java [hunk 3]
```diff
@@ -209,6 +209,11 @@ public void blockingLastEmpty() {
         Observable.empty().blockingLast();
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void blockingIterableNegativeBufferSizeFail() {
```
Comment (akarnokd) [inline src/test/java/io/reactivex/internal/operators/observable/ObservableBlockingTest.java]: This is covered by the [ParamValidationCheckerTest](https://github.com/ReactiveX/RxJava/blob/2.x/src/test/java/io/reactivex/ParamValidationCheckerTest.java) class which scans all operators for parameter validation.

File: src/test/java/io/reactivex/schedulers/TimedTest.java [hunk 4]
```diff
@@ -84,4 +84,10 @@ public void toStringOf() {
 
         assertEquals("Timed[time=5, unit=SECONDS, value=1]", t1.toString());
     }
+
+    @SuppressWarnings({"unchecked", "rawtypes"})
```
Comment (akarnokd) [inline src/test/java/io/reactivex/schedulers/TimedTest.java]: There is no need to suppress this.

File: src/test/java/io/reactivex/schedulers/TimedTest.java [hunk 5]
```diff
@@ -84,4 +84,10 @@ public void toStringOf() {
...
+    @Test(expected = NullPointerException.class)
+    public void timeUnitNullFail() throws Exception {
+        Timed<Integer> t1 = new Timed<Integer>(1, 5, null);
```
Comment (akarnokd) [inline src/test/java/io/reactivex/schedulers/TimedTest.java]: `t1` shows up as unused variable, please remove the assignment.

**Review by akarnokd — COMMENTED** [review]
Some adjustments are still needed, otherwise :+1:

File: src/test/java/io/reactivex/internal/operators/observable/ObservableBlockingTest.java (hunk 3 above)
Comment (ggikko) [reply src/test/java/io/reactivex/internal/operators/observable/ObservableBlockingTest.java]: Oh.. yeap I got it. thank you 👍

File: src/test/java/io/reactivex/schedulers/TimedTest.java (hunk 4 above)
Comment (ggikko) [reply src/test/java/io/reactivex/schedulers/TimedTest.java]: yeap I got it. thank you 👍 I removed this annotation.

File: src/test/java/io/reactivex/schedulers/TimedTest.java (hunk 5 above)
Comment (ggikko) [reply src/test/java/io/reactivex/schedulers/TimedTest.java]: I agree and removed useless variable.

**Pull request merged**